spec/grid_spec.rb

```diff
@@ -73,14 +73,23 @@
       expect(grid.coordinate_valid?(invalid_lower_case)).to eq false
     end
   end
-  describe "#mark" do
-    it "takes in a valid coordinate, and gives it a marked value in coordinates" do
-      coordinate = "A3"
-      grid = Grid.new
 
-      grid.mark(coordinate)
+  it "#mark takes in a valid coordinate, and gives it a marked value in coordinates" do
+    coordinate = "A3"
+    value = 'X'
+    grid = Grid.new
 
-      expect(grid.get_value(coordinate)).to eq('X')
-    end
+    grid.mark(coordinate, value)
+
+    expect(grid.get_value(coordinate)).to eq('X')
+  end
+
+  it "#get_computer_choice returns a random key from the structure" do
```

Comment:
I'm not sure what this test accomplishes. Maybe this is a place where you can use `srand` again to assure the value is always the same it tests.

Reply:
Ah, good point. ✔️